records/generator.go

```diff
@@ -176,12 +176,12 @@ func (rg *RecordGenerator) loadFromMaster(ip string, port string) (sj StateJSON)
 	if err != nil {
 		logging.Error.Println(err)
 	}
-	defer resp.Body.Close()
 
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		logging.Error.Println(err)
 	}
+	_ = resp.Body.Close()
```

Comment:
the docs use `defer resp.Body.Close()`. not sure why this was changed. using `defer` provides "finally"-like semantics, which I think are appropriate in this case. If for some reason there's a panic that bubbles up we probably want to make sure that the Body is closed.